src/current/v23.1/connection-pooling.md

```diff
@@ -29,34 +29,90 @@ Connection pooling can be a enabled as a feature of the driver, a separate libra
 
 Creating the appropriate size pool of connections is critical to gaining maximum performance in an application. Too few connections in the pool will result in high latency as each operation waits for a connection to open up. But adding too many connections to the pool can also result in high latency as each connection thread is being run in parallel by the system. The time it takes for many threads to complete in parallel is typically higher than the time it takes a smaller number of threads to run sequentially.
 
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+In {{ site.data.products.serverless }} clusters, the purpose of the connection pool is to reduce connection latency, not to set a maximum number of simultaneous connections. {{ site.data.products.serverless }} clusters will automatically scale up to meet the demand from multiple simultaneous client connections.
+
+{% capture connection_timeout_value_serverless %}
+Configure the connection timeout to be at least 15 minutes and to not exceed 30 minutes.
+{% endcapture %}
+
+{{ connection_timeout_value_serverless }}
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="dedicated selfhosted">
+
 Each processor core can only execute one thread at a time. When there are more threads than processor cores, the system will use context switching to [time-slice](https://wikipedia.org/wiki/Preemption_(computing)#Time_slice) the thread execution. For example, if you have a system with a single core and two threads, processing threads 1 and 2 in parallel results in the system context switching to pause execution of thread 1 and begin executing thread 2, and then pause execution of thread 2 to resume executing thread 1. Executing thread 1 completely and then executing thread 2 will be faster because the system doesn't need to context switch, even though thread 2 had to wait until thread 1 fully completed to begin executing.
 
 Storage and network performance also will affect the ability of a thread to fully execute. If a thread is blocked by network or storage latency, adding connections to the pool is a good idea so other threads can execute while the original thread is being blocked.
 
 Cockroach Labs performed lab testing of various customer workloads and found no improvement in scalability beyond:
 
-**connections = (number of cores * 4)**
+**active connections = (number of cores * 4)**
```

Comment:
How is this actionable, though? Connection pools do not allow the # active connections to be limited, they only allow the # total connections to be limited. Furthermore, most apps have a number of worker nodes, each of which has its own connections to the database, which complicates things further.